### PR TITLE
Add PostgreSQL

### PR DESCRIPTION
**Pull request by FalseR20**
# How to test locally:
**Preparations:**
- poetry install
- create database in postgres as in readme
- change database system in [config.py](https://github.com/FalseR20/sc-server-auth/blob/feature/add_postgres/sc_server_auth/config.py#:~:text=%3A%20Database.-,SQLITE,-%2C%20%20%23%20Change%20database)
- run [database_default_insert.sql](https://github.com/FalseR20/sc-server-auth/blob/feature/add_postgres/database_default_insert.sql) in pycharm

**Testing:**
- local ci
- run server

### CODE REVIEW

File: sc_server_auth/server/database.py [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
-@event.listens_for(Engine, "connect")
-def set_sqlite_pragma(dbapi_connection, connection_record):
-    cursor = dbapi_connection.cursor()
-    cursor.execute("PRAGMA foreign_keys=ON")
-    cursor.close()
+# class Role(Base):
+#     __tablename__ = cnt.ROLE
+#     id = Column(Integer, primary_key=True, unique=True)
+#     name = Column(String(255), nullable=False)
+#     users = relationship(cnt.USER)
```
Comment (AlexandrZagorskiy) [inline sc_server_auth/server/database.py]: please remove comments

File: sc_server_auth/server/database.py [hunk 2]
```diff
@@ -33,11 +32,9 @@ def serialize(self):
 
 class DataBase:
     def __init__(self) -> None:
-        self.engine = create_engine(params[cnt.SQLITE_DB_PATH])
+        self.engine = db_engines[db_params[cnt.DATABASE]]()
         self.session = None
-
-    def init(self) -> None:
```
Comment (AlexandrZagorskiy) [inline sc_server_auth/server/database.py:39]: why you decide to delete init function? Is your connection to postgres didn't break the logic?

File: README.md [hunk 3]
```diff
@@ -21,6 +21,32 @@ Activate the virtual environment is to create a new shell with **_poetry shell_*
...
+grant all privileges on database sc_auth to sc_auth;
+```
+
+**Activation**
+
+Set postgres in `sc_server_auth/config.py:24:28`  <!--- temporally, config in progress -->
+
+**Insert default values**
+
+Run `database_default_insert.sql` in pycharm  <!--- temporally -->
```
Comment (AlexandrZagorskiy) [inline README.md]: Can you make a script which will execute this code automatically? Also I didn't like that you mention pycharm here, can you remove it?

**Review by AlexandrZagorskiy — CHANGES_REQUESTED** [review]

File: sc_server_auth/server/database.py (hunk 2 above)
Comment (FalseR20) [reply sc_server_auth/server/database.py:39]: def init() was not used and table was not created. I connected __init__ and init without breaking the logic

File: sc_server_auth/server/database.py (hunk 1 above)
Comment (FalseR20) [reply sc_server_auth/server/database.py]: done

File: README.md (hunk 3 above)
Comment (FalseR20) [reply README.md]: It will be in readme

**Review by AlexandrZagorskiy — APPROVED** [review]

**Pull request merged**